VideoWeb/VideoWeb.Common/SignalR/AdUserProfileService.cs

```diff
@@ -36,9 +47,19 @@ public async Task<string> GetObfuscatedUsernameAsync(string username)
 
         public async Task<UserProfile> GetUserAsync(string username)
         {
+            if (username.EndsWith(QuickLinkParticipantConst.Domain))
```

Comment:
create a private function here, thats takes username as an argument and then returns a boolean value, this is because this logic is duplicated here and in line 33